lib/node_modules/@stdlib/math/base/special/acoth/docs/repl.txt

```diff
@@ -0,0 +1,28 @@
+
+{{alias}}( x )
+    Computes the inverse hyperbolic cotangent of a number.
+
+    If `x >= -1 && x <= 1`, the function returns `NaN`.
+
+    Parameters
+    ----------
+    x: number
+        Input value.
+
+    Returns
+    -------
+    y: number|out
```

Comment:
```suggestion
    y: number
```

Reply:
I thought this was causing my linting error check because the function can return a number or NaN. I added it becasue the REPL documentation provides a case with multiple output values. Is `out` in this case the wrong return type or it should return just one type?

Reply:
`NaN` is a number value, despite its "not-a-number" name. More specifically, it is a particular set of bit sequences which do not have a decimal representation when read from a value stored in double-precision floating-point format. Accordingly, even if a function returns `NaN`, using the `number` type is still appropriate.

Additionally, in this case, `out` is not a value type, so would not be appropriate in any context for parameter or return value types in REPL help documentation or in JSDoc type annotations.

You are correct, some functions in the project return values of different "types", but this is not one of those cases. :)